missing-types.d.ts

```diff
@@ -39,7 +39,6 @@ interface Test {
 interface TestMeta {
   title: string;
   shortDesc: string;
-  importance: number;
```

Comment:
I've removed importance for now, since we're not using it. We can add it again later if we want it, and know how to represent it.